code/unclassified/biggest_of_n_numbers/biggest_of_n_numbers.py

```diff
@@ -0,0 +1,6 @@
+elements = []
+n = int(input("Enter number of elements:"))
+for i in range(1, n + 1):
+  elements.append(int(input("Enter element:")))
```

Comment:
Enter the next element: is a better prompt.

Reply:
It's running in loop so if i write  "Enter the next element:"
this would be shown for the first element too which is incorrect . 
So i think it is not better